src/browser/MsalProxy.js

```diff
@@ -43,6 +43,7 @@ function signInSilent(success, error, opts) {
                         account: {
                             id: resp.uniqueId,
                             username: resp.account.username,
+                            idToken: resp.idToken,
```

Comment:
This change needs to be replicated in `signInInteractive` as well

Reply:
Thanks, I missed that.